cdap-client/src/main/java/co/cask/cdap/client/ProgramClient.java

```diff
@@ -112,6 +120,23 @@ public void start(String appId, ProgramType programType, String programName, Map
    * @param appId ID of the application that the program belongs to
```

Comment:
Should change the method description to indicate why this method has been added. How about:
"Starts a program, giving debug mode, using the stored runtime arguments."